### ISSUE TITLE
Convert to Mp3 no longer supported ?

### ISSUE THREAD

**Issue by inconhito21** [openaudible/openaudible#229]
Hello, 

I just installed OpenAudible on my mac and I see the convert to mp3 field is gray. Is it no longer available, or what may the problem be ? I have my Audible account connected and my audiobooks synced but can't manage to convert them or listen to them. 

Thank you very much

![Capture d’écran 2019-11-19 à 17 53 59](https://user-images.githubusercontent.com/57958093/69167672-a2a1cf00-0af5-11ea-82ab-8b1ea366375d.png)

**Comment by BSCR** [comment]
This happens to mine if I have the check for download manager option selected in the Audible account settings.

Open the browser, login to your audible account, and under the account settings there is a 'Check for Audible Download Manager' option. This needs to be unticked. Then I found I needed to reload OpenAudible and it worked as required.

**Comment by fcastilloec** [comment]
@BSCR This fixed my issue, thanks! I think this should be closed.
Also @openaudible might want to add this info to the README of this project. The official website mentions:

> Note: If you get a message about changing an audible download manager setting, follow these directions.
But none of us received that message, so maybe the official website should be updated, or this is a related bug where a message should have been shown stating the problem and what to do.

**Comment by openaudible** [comment]
Its not something we test often.. but a long while ago it was working.. something probably changed on audible's end. Will try to address.

**Comment by inconhito21** [comment]
Awesome thanks it fixed perfectly!!



Envoyé depuis Yahoo Mail pour iPhone


Le jeudi, novembre 21, 2019, 9:38 AM, BSCR <notifications@github.com> a écrit :


This happens to mine if I have the check for download manager option selected in the Audible account settings.

Open the browser, login to your audible account, and under the account settings there is a 'Check for Audible Download Manager' option. This needs to be unticked. Then I found I needed to reload OpenAudible and it worked as required.
